### DISCUSSION TITLE
Matterbridge Zigbee2MQTT plugin pulling wrong path

### DISCUSSION THREAD

**Discussion by squito1** [tteck/Proxmox#3544] [Help Needed] [answered]
Hello, I reached out to Luligu on this and he directed me back to the LXC code. When I try to enable the Z2M plugin i get errored out with the code below. I will put Luligu's answer also below. In his answer, I had to fix some at my end also which I have done (removed the http://). Please let me know if you need anything further. Thank you.

squito1 commented 20 hours ago
I have successfully deployed Matterbridge in a ProxMox LXC and Matterbridge is located in a browser. I have installed the Zigbee2MQTT also in an LXC successfully but when I try to get the two to connect I get the error message below. I have used the IP for the Zigbee2MQTT for the host along with my username and password. Please assist if you can. Thank you.

Plugins - Enabling plugin: matterbridge-zigbee2mqtt
info - [12:50:34.481] [PluginManager]: Enabled plugin matterbridge-zigbee2mqtt
info - [12:50:34.484] [PluginManager]: Loading plugin matterbridge-zigbee2mqtt type DynamicPlatform
info - [12:50:34.628] [Matterbridge zigbee2mqtt plugin]: Initializing platform: matterbridge-zigbee2mqtt v
info - [12:50:34.629] [Matterbridge zigbee2mqtt plugin]: Loaded zigbee2mqtt parameters from /root/.matterbridge/matterbridge-zigbee2mqtt.config.json:
info - [12:50:34.631] [Matterbridge zigbee2mqtt plugin]: Connecting to MQTT broker: mqtt://http://192.168.1.25:1883
error - [12:50:34.772] [Zigbee2MQTT]: Error connecting to mqtt://http://192.168.1.25:1883: getaddrinfo ENOTFOUND http
error - [12:50:34.773] [Matterbridge zigbee2mqtt plugin]: MQTT broker at http://192.168.1.25:1883 error: Error: getaddrinfo ENOTFOUND http
notice - [12:50:34.778] [PluginManager]: Loaded plugin matterbridge-zigbee2mqtt type DynamicPlatform (entrypoint /usr/lib/node_modules/matterbridge-zigbee2mqtt/dist/index.js)
info - [12:50:34.779] [PluginManager]: Starting plugin matterbridge-zigbee2mqtt type DynamicPlatform
info - [12:50:34.780] [Matterbridge zigbee2mqtt plugin]: Starting zigbee2mqtt dynamic platform v2.1.10: The plugin has been enabled
error - [12:50:49.781] [Matterbridge zigbee2mqtt plugin]: The plugin did not receive zigbee2mqtt bridge state. Check if zigbee2mqtt is running and connected to the MQTT broker.
error - [12:50:49.782] [Matterbridge zigbee2mqtt plugin]: The plugin did not receive zigbee2mqtt bridge info. Check if zigbee2mqtt is running and connected to the MQTT broker.
error - [12:50:49.782] [Matterbridge zigbee2mqtt plugin]: The plugin did not receive zigbee2mqtt bridge devices/groups. Check if zigbee2mqtt is running and connected to the MQTT broker.
error - [12:50:49.782] [PluginManager]: Failed to start plugin matterbridge-zigbee2mqtt: Error: The plugin did not receive zigbee2mqtt bridge state or info or devices/groups. Check if zigbee2mqtt is running and connected to the MQTT broker.
error - [12:50:49.783] [PluginManager]: Plugin matterbridge-zigbee2mqtt not started.

Luligu commented 7 hours ago • 
Hi,
I see this:

Connecting to MQTT broker: mqtt://http://192.168.1.25:1883

Put only the ip in the config.

For LXC, I hope they updated the script...
let us know please.

**Comment by tteck** [comment, marked as answer]
It appears that you have entered the MQTT host IP incorrectly.

![Screenshot 2024-08-18 1 23 44 PM](https://github.com/user-attachments/assets/d9081668-6345-4b00-ae85-a9e405c98d0c)

**Reply by tteck** [reply]
![Screenshot 2024-08-18 1 47 33 PM](https://github.com/user-attachments/assets/4decaa83-1dd0-44a9-b4e2-e8572764fd07)

**Comment by squito1** [comment]
Hello,
Well, I was able to fix it. I changed the IP as suggested but realized that it needed to be the MQTT address and not the Z2M address. It is all working now. Thank you for your help.
